### PR TITLE
Disabilities transportation question

### PR DESCRIPTION
**Pull request by MariaGarciaCS**
Finished first table. There's a couple more I want to add and a bar chart too, but this can be displayed on the site as it it rn.

### CODE REVIEW

File: app/site/questions.js [hunk 1]
```diff
@@ -6,7 +6,11 @@ import ResidentsWithDisabilities from '../components/questions/ResidentsWithDisa
 import BelongingRates from '../components/questions/BelongingRates'
 import RentBurdenedRates from '../components/questions/RentBurdenRates'
 import SidewalkCafePermitsYears from '../components/questions/SidewalkCafePermitsYears'
+<<<<<<< HEAD
+import DisabilitiesAndTransportation from '../components/questions/DisabilitiesAndTransportation'
+=======
```
Comment (vingkan) [inline app/site/questions.js:11]: There are some merge markers in this file, make sure you resolve the merge conflicts!

**Review by vingkan — CHANGES_REQUESTED** [review]
Awesome! Can you fix the merge conflicts and add a screenshot of how the page looks in your PR description for documentation?

There is a lot of SQL logic to review and I don't want to block you, so if the site builds and the page works, we can approve this.

Try clicking on the column names to sort them to see whether or not that causes an error. If you can sort all the columns without errors that'll be good.

File: app/site/questions.js (hunk 1 above)
Comment (MariaGarciaCS) [reply app/site/questions.js:11]: image

**Review by vingkan — APPROVED** [review]
Excellent, Maria! This is a great foundation for your data question. Glad we could get the work you've been doing checked in.

**Pull request merged**